sunpy/net/jsoc/attrs.py

```diff
@@ -110,9 +110,11 @@ class Notify(SimpleAttr):
 
     def __init__(self, value):
         super().__init__(value)
+        if not value:
```

Comment:
```suggestion
        if value is None:
```
Is a bit clearer (if `value` is indeed `None` if no email is passed...